src/Service.Tests/SqlTests/GraphQLMutationTests/GraphQLMutationTestBase.cs

```diff
@@ -1111,6 +1113,35 @@ public virtual async Task TestParallelDeleteMutations()
 
         }
 
+        /// <summary>
+        /// Performs create item on different Windows Regional format settings and validate that the data type Float is correct
+        /// </summary>
+        public async Task CanCreateItemWithCultureInvariant(string cultureInfo, string dbQuery)
+        {
+            CultureInfo ci = new(cultureInfo);
+            CultureInfo.DefaultThreadCurrentCulture = ci;
+
+            string graphQLMutationName = "createSales";
+            string graphQLMutation = @"
+                mutation {
+                    createSales (item: { item_name: ""test_name"", subtotal: 3.14, tax: 1.15 }) {
+                        id
+                        item_name
+                        subtotal
+                        tax
+                        total
```

Comment:
how is `total` calculated?

Reply:
`total` is calculated by adding `subtotal` and `tax`.